apps/controlled-drugs/fields/index.js

```diff
@@ -302,5 +302,39 @@ module.exports = {
   'person-responsible-for-security-confirmed-dbs': {
     mixin: 'checkbox',
     validate: ['required']
+  },
+  'is-employee-or-consultant': {
+    mixin: 'radio-group',
+    isPageHeading: true,
+    validate: [ 'required' ],
+    options: [
+      {
+        value: 'employee'
+      },
+      {
+        value: 'consultant'
+      }
+    ],
+    className: ['govuk-radios', 'govuk-radios--inline'],
+    legend: {
+      className: 'govuk-!-margin-bottom-6'
+    }
+  },
+  'has-witness-destruction-of-drugs': {
```

Comment:
I would suggest a slight renaming to give it a more relevant meaning
```suggestion
  'require-witness-destruction-of-drugs': {
```